util/cron/nightly

```diff
@@ -242,8 +245,7 @@ if ($printusage == 1) {
 
 print "\n";
 system("printenv");
-$modulecmd = "/sw/modules/bin/linux/modulecmd"; # heuristic
-if (-x $modulecmd) { system("$modulecmd bash list"); }
+system("module list");
```

Comment:
We probably need to check if the `module` command is available first

Reply:
`module list` just quietly fails if it doesn't work and things proceed without printing the list. Where we used to get the error about loading shared libraries, it's just a blank line with this.